my_portfolio/resources/js/script.js

```diff
@@ -0,0 +1,43 @@
+const menuBtn = document.querySelector('.menu-btn');
+const reaction = document.querySelector('.menu-btn_options');
+const menuNav = document.querySelector('.menu-nav');
+const navItems = document.querySelectorAll('.menu-nav__item');
+
+let showMenu = false;
+
+function toggleMenu() {
+  if (!showMenu) {
+    reaction.classList.add('open');
+    menuNav.classList.add('open');
+    navItems.forEach((item) => item.classList.add('open'));
+
+    showMenu = true;
+  } else {
+    reaction.classList.remove('open');
+    menuNav.classList.remove('open');
+    navItems.forEach((item) => item.classList.remove('open'));
+
+    showMenu = false;
+  }
+}
+
+menuBtn.addEventListener('click', toggleMenu);
+
+const sections = document.querySelectorAll('section');
+const navLinks = document.querySelectorAll('header nav a');
+
```

Comment:
- Is it possible to move all define variables to the top? if yes! pls do.

Reply:
Ok sir